src/components/FrontPage/components/NewestEpisodes/styles.scss

```diff
@@ -2,7 +2,7 @@
 
 .container  {
   position: relative;
-  display: block;
```

Comment:
makes the images go to the center of the box if the title on the article next to it is longer than the newestEpisode container

Reply:
Aha, I see 👍 
 GJ!